Add env vars for excluding OPW logs (#28409)
DataDog/documentation: content/en/observability_pipelines/sources/datadog_agent.md

Use the Agent configuration file or the Agent Helm chart values file to connect the Datadog Agent to the Observability Pipelines Worker.

> **Note**: If your Agent is running in a Docker container, you need to exclude Observability Pipelines logs using the `DD_CONTAINER_EXCLUDE_LOGS` environment variable. For Helm, use `datadog.containerExcludeLogs`. Otherwise, you get duplicate Observability Pipelines logs because the Worker also sends logs to Datadog. See [Docker Log Collection][2] or [Setting environment variables for Helm][3] for more information.

Reviewer: tiny nit - just to clarify we're talking about the worker's internal logs in this case, feel free to adjust 
```suggestion
**Note**: If your Agent is running in a Docker container, you need to exclude Observability Pipelines logs using the `DD_CONTAINER_EXCLUDE_LOGS` environment variable. For Helm, use `datadog.containerExcludeLogs`. Otherwise, you get duplicate Observability Pipelines logs because the Worker also sends its own logs directly to Datadog. See [Docker Log Collection][2] or [Setting environment variables for Helm][3] for more information.
```

> **Note**: If your Agent is running in a Docker container, you need to exclude Observability Pipelines logs using the `DD_CONTAINER_EXCLUDE_LOGS` environment variable. For Helm, use `datadog.containerExcludeLogs`. Otherwise, you get duplicate Observability Pipelines logs because the Worker also sends its own logs directly to Datadog. See [Docker Log Collection][2] or [Setting environment variables for Helm][3] for more information.

Reviewer 2: ```suggestion
**Note**: If your Agent is running in a Docker container, you must exclude Observability Pipelines logs using the `DD_CONTAINER_EXCLUDE_LOGS` environment variable. For Helm, use `datadog.containerExcludeLogs`. This prevents duplicate logs, as the Worker also sends its own logs directly to Datadog. See [Docker Log Collection][2] or [Setting environment variables for Helm][3] for more information.
```
